cmd/ethrex/Cargo.toml

```diff
@@ -40,6 +40,10 @@ ethrex-metrics = { path = "../../crates/blockchain/metrics" }
 name = "ethrex"
 path = "./ethrex.rs"
 
+[lib]
+name = "ethrex"
+path = "./ethrex.rs"
+
```

Comment:
Let's create a `lib.rs`, move the modules definition there and use that as the library we'll expose. This wy `ethrex.rs` remains like a `main.rs`, "user" of `lib.rs`.

Reply:
Changed in [6770eb4](https://github.com/lambdaclass/ethrex/pull/2190/commits/6770eb4a9bb1b94b391754af96a348827b44b277) and [59aa8d2](https://github.com/lambdaclass/ethrex/pull/2190/commits/59aa8d2809ca315e491815ae4a602fa080c5809f)